### PR TITLE
FluidDataSetWr example code

### PR DESCRIPTION
**Pull request by tedmoore**

### CODE REVIEW

**Review by jamesb93 — APPROVED** [review]

File: release-packaging/HelpSource/Classes/FluidDataSetWr.schelp [hunk 1]
```diff
@@ -132,18 +112,24 @@ OSCFunc({
 ~ds.print;
 ~ds.clear
 
-// circular writing
+::
+strong::circular writing::
+Each time link::Classes/FluidDataSetWr:: is triggered it is like the link::Classes/FluidDataSet#-setPoint:: method so if the identifier does not exist it creates it. If the identifier does it exist then it updates it with the new values.
+
+By looping code::idNumber:: values, we can use a link::Classes/FluidDataSet:: similar to a "circle buffer", always have the most recent code::n:: points in it that we want.
```
Comment (weefuzzy) [inline release-packaging/HelpSource/Classes/FluidDataSetWr.schelp:119]: *circular* buffer

File: release-packaging/HelpSource/Classes/FluidDataSetWr.schelp [hunk 2]
```diff
@@ -132,18 +112,24 @@ OSCFunc({
 ~ds.print;
 ~ds.clear
 
-// circular writing
+::
+strong::circular writing::
+Each time link::Classes/FluidDataSetWr:: is triggered it is like the link::Classes/FluidDataSet#-setPoint:: method so if the identifier does not exist it creates it. If the identifier does it exist then it updates it with the new values.
```
Comment (weefuzzy) [inline release-packaging/HelpSource/Classes/FluidDataSetWr.schelp:117]: Too many `it`s! Suggest 

'so if the identifier does not exist, it creates a new point. If the identifier does exist, the corresponding point is updated'

File: release-packaging/HelpSource/Classes/FluidDataSetWr.schelp [hunk 3]
```diff
@@ -67,58 +67,38 @@ s.reboot;
...
-    var idx = Stepper.kr(trig,min:-1, max:n); //we need to start at -1 to catch the first increment
-	4.collect{|i| BufWr.kr([(4 * idx) + i],b,i)};
-    FluidDataSetWr.kr(~ds,idNumber:idx,buf:b,trig:trig);
-    SendReply.kr(idx >= (n-1), '/datasetwrdone');
-    FreeSelf.kr(idx >= (n-1));
-}.play(s,args:[n:100]);
+{
+	arg n;
+	var buf = LocalBuf(4);
+	var trig = Impulse.kr(ControlRate.ir / 8); // can't go any faster
```
Comment (weefuzzy) [inline release-packaging/HelpSource/Classes/FluidDataSetWr.schelp:77]: Why can't we go any faster? Is this empirical or is there a logical explanation (I would have thought `ControlRate.ir / 2` would be more likely)

**Comment by weefuzzy** [comment]
Oh, I see it was already merged. Ah, well.

**Pull request merged**